### ISSUE TITLE
Debug Solutions to Python Challenges

### ISSUE THREAD

**Issue by teenageknight** [teenageknight/Python-Beginner-Projects#4]
There are many errors in the solutions for the challenges because this project was taken from an old repository that used python 2.7. Updated solutions are required.

**Comment by AliHaider20** [comment]
I have found some issues in the code and there is are some space for improvement can you assign this to me?

**Comment by teenageknight** [comment]
Absolutely! I have kind of abandoned this project to work on some of the other ones, but any changes you want to make feel free.

**Comment by AliHaider20** [comment]
Can you explain the structure of 100+ problems. Why have you made a question files folder when the solution is given in the text. Even the solutions are the same.

**Comment by teenageknight** [comment]
Oh my bad. It has been a while since I looked at this. I am fairly certain I got the text file from somewhere online. I just did the questions in their own python file so I could test it and then compared that with the text file. TBH, I am not sure what changes you would even need to make. I don't plan on using it again and I think it is taken from someone else anyways so it is not like it needs to be super public-facing so other people can use it. It is more of a way for me to document the things I used to practice when I was starting out. If you see any obvious changes, or you want to create a file structure that is more readable, feel free, but I don't have any preferences/ wants right now.

**Comment by AliHaider20** [comment]
Ok np. So I'll have a look at the codes and there is some space for improvements. I'll make some PR's in few days. My request is to merge them.

**Comment by AliHaider20** [comment]
Hey I have create some PR's plz commit them.
